pkg/driverbuilder/builder/local.go

```diff
@@ -63,12 +63,14 @@ func (l *LocalBuilder) TemplateData(c Config, kr kernelrelease.KernelRelease, _
 				c.DriverVersion,
 				c.DeviceName,
 				c.DeviceName,
+				l.GccPath,
 				c.DriverVersion),
 		},
 		UseDKMS:       l.UseDKMS,
 		DownloadSrc:   len(l.SrcDir) == 0, // if no srcdir is provided, download src!
 		DriverVersion: c.DriverVersion,
 		KernelRelease: c.KernelRelease,
+		GCCPath:       l.GccPath,
```

Comment:
When building through `dkms`, cmake is not involved, therefore for `local` script, it must still be passed somehow.